gnosis/gp-swap-ui
# [ETH/WETH] Show warning if user native funds low

Part of #499

Shows error if user's native balance is lower than the static `0.1` threshold set in constants

### Styling by @biocom in #497 will make this look better

too low input
![Screenshot from 2021-04-23 11-20-32](https://user-images.githubusercontent.com/21335563/115857763-01bfed80-a426-11eb-9105-b5c3fbd09afa.png)

difference would leave too little
![Screenshot from 2021-04-23 12-37-58](https://user-images.githubusercontent.com/21335563/115865846-c971dc80-a430-11eb-9b9d-e1d0371c830d.png)

QA Testing
1. select an account with lessThan 0.1 in native token (ETH/XDAI)
2. select ETH SELL
3. select erc20 BUY
4. input an amount lessThan native balance
5. Error should show with wrapping component
